druid/src/widget/split.rs

```diff
@@ -0,0 +1,327 @@
+                }
+                let child1_width = reduced_width * self.split_point;
+                let child2_width = reduced_width - child1_width;
+                (
+                    BoxConstraints::new(
+                        Size::new(child1_width, bc.min.height),
+                        Size::new(child1_width, bc.max.height),
+                    ),
+                    BoxConstraints::new(
+                        Size::new(child2_width, bc.min.height),
+                        Size::new(child2_width, bc.max.height),
+                    ),
+                )
+            }
+            Axis::Vertical => {
+                if !bc.is_width_bounded() {
+                    log::warn!("A Split widget was given an unbounded height to split.")
+                }
+                let child1_height = reduced_height * self.split_point;
+                let child2_height = reduced_height - child1_height;
+                (
+                    BoxConstraints::new(
+                        Size::new(bc.min.width, child1_height),
+                        Size::new(bc.max.width, child1_height),
+                    ),
+                    BoxConstraints::new(
+                        Size::new(bc.min.width, child2_height),
+                        Size::new(bc.max.width, child2_height),
+                    ),
+                )
+            }
+        };
+        let child1_size = self.child1.layout(ctx, &child1_bc, &data, env);
+        let child2_size = self.child2.layout(ctx, &child2_bc, &data, env);
+
+        //Top-left align for both children, out of laziness.
+        //Reduce our unsplit direction to the larger of the two widgets
+        let (child1_rect, child2_rect) = match self.split_direction {
+            Axis::Horizontal => {
+                my_size.height = child1_size.height.max(child2_size.height);
+                (
+                    Rect::from_origin_size(Point::ORIGIN, child1_size),
+                    Rect::from_origin_size(
+                        Point::new(child1_size.width + self.splitter_size, 0.0),
+                        child2_size,
+                    ),
+                )
+            }
+            Axis::Vertical => {
+                my_size.width = child1_size.width.max(child2_size.width);
+                (
+                    Rect::from_origin_size(Point::ORIGIN, child1_size),
+                    Rect::from_origin_size(
+                        Point::new(0.0, child1_size.height + self.splitter_size),
+                        child2_size,
+                    ),
+                )
+            }
+        };
+        self.child1.set_layout_rect(child1_rect);
+        self.child2.set_layout_rect(child2_rect);
+        my_size
+    }
+
+    fn event(&mut self, event: &Event, ctx: &mut EventCtx, data: &mut T, env: &Env) {
+        if self.draggable && self.splitter_size > 0.0 {
+            //TODO: Draggable splitter
```

Comment:
can get rid of this todo?

Reply:
😅 It shouldn't have made it to this commit in the first place.